src/store.js

```diff
@@ -144,18 +144,32 @@ export class Store {
       ? Promise.all(entry.map(handler => handler(payload)))
       : entry[0](payload)
 
-    return result.then(res => {
-      try {
-        this._actionSubscribers
-          .filter(sub => sub.after)
-          .forEach(sub => sub.after(action, this.state))
-      } catch (e) {
-        if (process.env.NODE_ENV !== 'production') {
-          console.warn(`[vuex] error in after action subscribers: `)
-          console.error(e)
+    return new Promise((resolve, reject) => {
+      result.then(res => {
+        try {
+          this._actionSubscribers
+            .filter(sub => sub.after)
+            .forEach(sub => sub.after(action, this.state))
+        } catch (e) {
+          if (process.env.NODE_ENV !== 'production') {
```

Comment:
Please change `process.env.NODE_ENV !== 'production'` to `__DEV__`. This change was introduced in the latest commit.

```suggestion
          if (__DEV__) {
```